docs/examples/tests/data_transfer_objects/test_dto_model_interaction.py

```diff
@@ -11,16 +13,18 @@
 )
 
 
+@pytest.mark.xfail(reason="sqlalchemy plugin model map returns wrong instance for test")
 def test_dto_from_model_instance() -> None:
     assert isinstance(dto_instance, FromModelCompanyDTO)
 
     for field in FromModelCompanyDTO.__fields__:
         assert getattr(dto_instance, field) == getattr(company_instance, field)
 
 
+@pytest.mark.xfail(reason="sqlalchemy plugin model map returns wrong instance for test")
```

Comment:
This test might still be failing as I haven't seen any xpassed but if it is, it isn't for this reason anymore.